mkroosevelt.js

```diff
@@ -45,7 +51,14 @@ else if (cmd === '-v' || cmd === '--v' || cmd === '-version' || cmd === '--versi
 }
 else if (cmd) {
   try {
-    createSampleApp();
+    if (cmd === '.') {
+      currentDirectory = true;
+      createSampleApp(currentDirectory);
```

Comment:
Ah yes it will merge the files, didn't know you were asking that. Still want the warning then?